src/pages/Schedules/Schedules.tsx

```diff
@@ -8,22 +9,45 @@ import SubTitle from '../../components/SubTitle/SubTitle'
 import AppContext from '../../Context/App.context'
 import StepBar from '../../components/StepBar/StepBar'
 import ArrowButton from '../../components/Button/ArrowButton'
+import { RootState } from '../../store'
+import { isWeekend, parse } from 'date-fns'
 
 export default function Schedules() {
-  const { setTime, setCurrentStep, time, setPage } = useContext(AppContext)
-  const buttons = [{ label: "20:00", value: 20 },
-  { label: "21:00", value: 21 },
-  { label: "19:00", value: 19 },
-  { label: "18:00", value: 18 },
-  { label: "17:00", value: 17 },
-  { label: "16:00", value: 16 }]
+  const { setTime, setCurrentStep, time, setPage, date, duration } = useContext(AppContext)
+  const [buttons, setButtons] = useState([{ label: '20', value: '0' }]);
```

Comment:
Why the label: '20'?

Reply:
Just dummy data, while we don't implement error messages.